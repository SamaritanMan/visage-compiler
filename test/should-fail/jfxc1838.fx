/*
 * Regression test JFXC-1838 :  incorrect type inference
 *
 * @test
 * @compile/fail
 */

class Foo {
    var str="Hello";
    var to;
}

var hvf="World";
var foo = Foo {
    to:bind hvf with inverse;
}