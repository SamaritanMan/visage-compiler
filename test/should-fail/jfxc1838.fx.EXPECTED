test/should-fail/jfxc1838.fx:15: Type mismatch for bind with inverse
found: String
required: Object
    to:bind hvf with inverse;
            ^
1 error
